packages/markdown/src/components/MarkdownBottomBar.tsx

```diff
@@ -33,20 +36,56 @@ export function MarkdownCounter(props: { words: number; characters: number }) {
   );
 }
 
-export function MarkdownHelp(props: { onClick: () => void }) {
+function SanitizeMessage() {
   return (
-    <Paragraph marginBottom="none" className={styles.help}>
+    <span>
+      The output of this markdown editor is sanitized.{' '}
```

Comment:
Please double-check the copy here with design. We need to somehow convey that this only affects the preview and not the content of the markdown editor.

Reply:
Done! It is now: `The preview of the content in this field will be sanitized. Learn more.`